packages/react-centra-checkout/src/Context/index.tsx

```diff
@@ -31,6 +31,12 @@ export interface ProviderProps {
     @defaultValue `centra-checkout-token`
   */
   tokenName?: string
+  tokenCookieOptions?: {
+    path?: string
+    domain?: string
+    secure?: boolean
+    sameSite?: 'Lax' | 'Strict' | 'None'
+  }
```

Comment:
Can we change this to be:
`tokenCookieOptions?: Cookies.CookieAttributes`

You can import the Cookie type like:
`import type Cookies from 'js-cookie'`

Reply:
Yes of course. Makes a lot of sense.